web/src/test/java/no/nav/modiapersonoversikt/rest/person/PersonsokControllerTest.kt

```diff
@@ -174,6 +173,79 @@ class PersonsokControllerTest {
                     ),
                     bostedsadresse = listOf(
                         SokPerson.Bostedsadresse(
+                            gyldigTilOgMed = null,
+                            matrikkeladresse = SokPerson.Matrikkeladresse(
+                                bruksenhetsnummer = "123101",
+                                tilleggsnavn = "Supergården",
+                                postnummer = "1234",
+                                kommunenummer = "654321"
+                            ),
+                            vegadresse = null,
+                            utenlandskAdresse = null,
+                            ukjentBosted = null
+                        )
+                    )
+                )
+            )
+
+            snapshot.assertMatches(lagPersonResponse(person))
+        }
+
+        @Test
+        internal fun `should map pdl response without address`() {
```

Comment:
Kunne den hatt et annet navn? Feks `should map pdl response when address is invalid`, `should filter out invalid address(es)`, `should not map invalid address` eller noe sånt